### PR TITLE
Build Optimizations

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by amilajack** [comment]
@dustintownsend for some reason, I cant add you as a reviewer:

screen shot 2016-12-14 at 10 48 30 pm

Guess I have to comment here to let you know to review this. thanks!

**Comment by amilajack** [comment]
LOL i just noticed that GitHub now has both contributors and collaborators. @chentsulin would be great if you could add @dustintownsend as a collaborator. He's done a lot for this project.

File: .babelrc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
-  "presets": ["es2015", "stage-0", "react"],
+	"presets": [
+    ["env", {"targets": { "electron": 1.4 }, "useBuiltIns": true }],
```
Comment (jhen0409) [inline .babelrc]: I think we should use `"node": 6` target, we need use for `build`, `test` and `hot-server`, make sure that node and electron will be different in the future, although currently they are the same. @amilajack What do you think?

**Comment by dustintownsend** [comment]
The babili-webpack-plugin works good. I just updated my project to use it as a test and it removed all those ugilifyjs warnings/errors I was getting!

File: .babelrc (hunk 1 above)
Comment (amilajack) [inline .babelrc]: @jhen0409 makes sense. At the moment, I believe node 6 and 7 have the exact same es6 compatibility.

**Comment by chentsulin** [comment]
Ok, I Already sent collaborator invitation to @dustintownsend.

**Comment by amilajack** [comment]
@jhen0409 Made all the changes you requested. Anything else?

**Pull request merged**